### PR TITLE
add info on mbip-5 to the key differences section

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds MBIP-5 to the key differences section of the transaction fees page

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/get-started/eth-compare/tx-fees.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ And then `TransactionWeight` is mapped to the following field of the block JSON
 extrinsics[extrinsic_number].events[event_number].data[0].weight
 ```
 
-### Key Differences with Ethereum {: #ethereum-api-transaction-fees}
+### Key Differences with Ethereum {: #key-differences-with-ethereum}
```
Comment (albertov19) [inline builders/get-started/eth-compare/tx-fees.md:214]: I feel that this section should be at the top of the page as an overall Summary. Also MBIP-5 Should have its own section where we show a clear difference from Ethereum. Maybe do a simple contract example where there is a clear contrast in gas estimation on Ethereum and on Moonbeam?

Or maybe  a contract that does provide the same exact gas estiation for a function, but then one very dfferent for another function?

**Review by albertov19 — CHANGES_REQUESTED** [review]
I think we need to add a bit more and reshuffle this section a bit 🙏

File: builders/get-started/eth-compare/tx-fees.md [hunk 2]
```diff
@@ -7,71 +7,185 @@ description: Learn about the transaction fee model used in Moonbeam and the diff
...
+
+  - Moonbeam has implemented a new mechanism defined in [MBIP-5](https://github.com/moonbeam-foundation/moonbeam/blob/master/MBIPS/MBIP-5.md){target=_blank} that limits block storage and increases gas usage for transactions that result in an increase in storage. Please note that this feature is only enabled on Moonbase Alpha at this time
+
+## Overview of MBIP-5 {: #overview-of-mbip-5 }
+
+MBIP-5 introduced changes to Moonbeam's fee mechanism that account for storage growth on the network, which deviates from the way Ethereum handles fees. By raising the gas needed to execute transactions that increase chain state and by establishing a block storage limit, it controls storage growth.
+
+This impacts contract deployments that add to the chain state, transactions that create new storage entries, and precompiled contract calls that result in the creation of new accounts.
+
+The block storage limit prevents transactions in a single block from collectively increasing the storage state by more than 40KB.
```
Comment (albertov19) [inline builders/get-started/eth-compare/tx-fees.md:32]: Set the block limit as a variable

File: builders/get-started/eth-compare/tx-fees.md [hunk 3]
```diff
@@ -7,71 +7,185 @@ description: Learn about the transaction fee model used in Moonbeam and the diff
...
+
+To determine the amount of gas for storage in bytes, there is a ratio that is defined as:
 
 ```text
-GET /blocks/{blockId}
+Ratio = Block Gas Limit / (Block Storage Limit * 1024 Bytes)
 ```
 
-The block endpoints will return data relevant to one or more blocks. You can read more about the block endpoints on the [official Sidecar documentation](https://paritytech.github.io/substrate-api-sidecar/dist/#operations-tag-blocks){target=_blank}. Read as a JSON object, the relevant nesting structure is as follows:  
+Given the block gas limit of 15,000,000 and the block storage limit of 40KB per block, the ratio between gas and storage is 366, which is calculated like this:
```
Comment (albertov19) [inline builders/get-started/eth-compare/tx-fees.md:40]: Use variables, I think it is more future-proof

**Review by albertov19 — APPROVED** [review]
LGTM!

Great job on the example it was SUPER clear and easy to follow.

Left a couple of minor suggestions

**Pull request merged**